.idea/Squadory.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="jdk" jdkName="Python 3.5.2 virtualenv at ~/.virtualenvs/djangodev" jdkType="Python SDK" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+  <component name="TestRunnerService">
+    <option name="PROJECT_TEST_RUNNER" value="Unittests" />
+  </component>
+</module>
```

Comment:
Please add `.idea` file in .gitignore.

Why?
- These files are IDE specific and each IDE will have it's own idea files for IDE.

You can use https://www.gitignore.io/api/python%2Cpycharm as gitignore file generated from gitignore.io

Reply:
Done! 
Edited the gitignore file to include .idea file.